docs/docusaurus.config.js

```diff
@@ -82,16 +88,23 @@ const config = {
                     },
                     {
                         type: "docSidebar",
-                        sidebarId: "gettingStartedSidebar",
+                        sidebarId: "examplesSidebar",
                         position: "left",
-                        label: "Getting started",
+                        label: "Examples",
```

Comment:
Shouldn't this be

```suggestion
                        label: "Guides",
```

?

Reply:
I think it's actually better that "guides" are a subsection of "examples" in general. There will be some examples that are not fully written out as walkthroughs with step-by-step instructions. If the navbar section header is Guides, then the less verbose examples will seem out of place.